tests/System/client_multivo.sh

```diff
@@ -0,0 +1,82 @@
+#!/bin/bash
+
+testdir=`echo $PWD`
+
+JDLFILE="${testdir}/multiVO.jdl"
+cat > ${JDLFILE} <<EOF
+[
+Executable = "multiVOexe.sh";
+StdOutput = "job.log";
+StdError = "job.log";
+InputSandbox = "multiVOexe.sh";
+OutputSandbox = "job.log";
+Site = "LCG.UKI-LT2-IC-HEP.uk";
+JobName = "MultiVOTest";
+]
+EOF
+
+
+EXEFILE="${testdir}/multiVOexe.sh"
+cat > ${EXEFILE} <<EOF
+echo $DIRACSITE
+ghostname=`hostname --long 2>&1`
+gipname=`hostname --ip-address 2>&1`
+echo $ghostname "has address" $gipname
+uname -a
+cat /etc/redhat-release
+EOF
+
+MYDATE=`date +%s`
```

Comment:
https://github.com/koalaman/shellcheck/wiki/SC2006
```suggestion
MYDATE=$(date +%s)
```